bodybank/character_generator/models.py

```diff
@@ -19,7 +19,25 @@ def __str__(self):
 
 
 class Movement(models.Model):
-    name = models.CharField(max_length=128)
+    WALKER = "Walker"
+    WINGED = "Winged"
+    SWIM = "Swim"
+    THRUST_VECTOR = "Thrust Vector"
+    HOPPER = "Hopper"
+    ROTOR = "Rotor"
+    IONIC = "Ionic"
+    WHEELED = "Wheeled"
+    MOVEMENT_NAME_CHOICES = [
+        (WALKER, 'Walker'),
```

Comment:
Seems you could use `(WALKER, WALKER), ...` this way you have the strings defined in one place instead of two